### ISSUE TITLE
MySmartRollerShades - Tilt App

### ISSUE THREAD

**Issue by mattlukc** [apexad/homebridge-mysmartblinds-bridge#16]
On the Tilt Smarter Home Website they also sell the MySmartRollerShades which required a different bridge and app (Tilt Smarthome). I was wondering if this is something you could get working with homebridge? Thanks!

**Comment by apexad** [comment]
Not at this time. I’ve got to work on issue #13 (switching this to a dynamic platform form static). Once that is complete, I will circle back to lol into this issue.

I do not have any eta on when they will be. Additionally without a tilt roller shade it may be difficult to troubleshoot this on my own, so when I am ready I may ask for your help.

**Comment by gregmichael** [comment]
Just updated to v2 and having issues with the "closing up" feature. It seems to be reversed. Will do some more testing on my end.

**Comment by apexad** [comment]
@gregmichael mind creating a new issue for this (and deleting your comment on this one). I see the problem and will fix it in a moment.
